app/locales/zh-TW/translation.json

```diff
@@ -0,0 +1,821 @@
+    "Arowell": "雅洛薇",
+    "Azalea": "亞潔理亞",
+    "Bask": "巴思克",
+    "Batisse": "巴托斯",
+    "Butcher Corps Inquisitor": "屠殺部隊員",
+    "Captain Rikoris": "先鋒隊長里科黎司",
+    "Carmainerose": "卡麥蘿茲",
+    "Carrot": "卡蘿",
+    "Celeste": "賽雷斯特",
+    "Chaos Inquisitor": "混沌教屠殺追擊者",
+    "Chaos Sect Axe": "混沌教巨斧大將軍",
+    "Church of Ilryos Axe": "伊利歐斯教斧兵",
+    "Commander Lorina": "指揮官蘿里娜",
+    "Doll Maker Pearlhorizon": "製偶師波蘿萊珍",
+    "Doris": "朵莉思",
+    "Eaton": "伊頓",
+    "Elson": "艾爾森",
+    "Enott": "艾諾特",
+    "Falconer Kluri": "鷹獵人可露莉",
+    "Glenn": "格倫",
+    "Gloomyrain": "格魯美蘭",
+    "Godmother": "教母",
+    "Gunther": "坤特",
+    "Hataan": "哈坦",
+    "Hazel": "海茲",
+    "Helga": "赫爾嘉",
+    "Hurado": "修拉杜",
+    "Ian": "伊安",
+    "Jecht": "傑克托",
+    "Jena": "捷娜",
+    "Judith": "茱迪絲",
+    "Kikirat v2": "奇奇拉特V.2",
+    "Kiris": "奇麗絲",
+    "Kluri": "可露莉",
+    "Lena": "雷娜",
+    "Lorina": "蘿里娜",
+    "Magic Scholar Doris": "魔法學者朵莉思",
+    "Mascot Hazel": "吉祥物海茲",
+    "Mercenary Helga": "自由自在的傭兵赫爾嘉",
+    "Mirsa": "米勒莎",
+    "Mistychain": "美絲緹彩",
+    "Montmorancy": "蒙茉朗西",
+    "Mucacha": "穆卡察",
+    "Nemunas": "尼姆拉斯",
+    "Otillie": "奧緹莉爾",
+    "Pearlhorizon": "波蘿萊珍",
+    "Pyllis": "費莉絲",
+    "Ras": "拉斯",
+    "Requiemroar": "雷奎姆洛",
+    "Researcher Carrot": "研究者卡蘿",
+    "Righteous Thief Roozid": "義賊魯茲德",
+    "Rikoris": "裏科黎司",
+    "Rima": "黎瑪",
+    "Roozid": "魯茲德",
+    "Sonia": "蘇尼婭",
+    "Sven": "史賓",
+    "Taranor Guard": "塔拉諾爾禁衛隊員",
+    "Taranor Royal Guard": "塔拉諾爾王宮士兵",
+    "Tieria": "黛莉亞",
+    "Wanda": "汪達",
+    "Zealot Carmainerose": "傳道者卡麥蘿茲",
+    "A Little Queen's Huge Crown": "小小女王的巨大王冠",
+    "Abyssal Crown": "深淵王冠",
+    "Alabastron": "雪花石膏瓶",
+    "Alencinox's Wrath": "艾蓮西諾克斯的逆鱗",
+    "Alexa's Basket": "雅莉莎的提籃",
+    "Ancient Dragon's Legacy": "古龍遺產",
+    "Bastion of Perlutia": "波魯迪亞的堡壘",
+    "Black Hand of the Goddess": "女神的黑手",
+    "Blood-Seared Moon": "血染之月",
+    "Bloodstone": "血玉髓",
+    "Bloody Rose": "血玫瑰",
+    "Border Coin": "孤註一擲",
+    "Celestine": "天青石",
+    "Champion's Trophy": "優勝者獎杯",
+    "Chatty": "柴迪",
+    "Circus Fantasia": "奇幻馬戲團",
+    "Cradle of Life": "生命搖籃",
+    "Creation & Destruction": "創造之福＆破壞之頌",
+    "Crimson Moon of Nightmares": "紅色噩夢之月",
+    "Crown of Glory": "榮耀之冠",
+    "Cruel Mischief": "殘酷的玩笑",
+    "Dignus Orb": "尊嚴寶珠",
+    "Doctor's Bag": "急救包",
+    "Double-Edged Decrescent": "雙刃下弦月",
+    "Draco Plate": "龍族板甲",
+    "Durandal": "杜蘭月",
+    "Dux Noctis": "暗夜指揮者",
+    "Elbris Ritual Sword": "埃布利斯的寶劍",
+    "Etica's Scepter": "艾帝嘉權杖",
+    "Fairy Tale for a Nightmare": "噩夢童話",
+    "Guiding Light": "引導之光",
+    "Holy Sacrifice": "神聖犧牲",
+    "Idol's Cheer": "歌姬助陣",
+    "Iron Fan": "鐵扇",
+    "Junkyard Dog": "Junkyard Dog",
+    "Justice for All": "為眾人的正義",
+    "Knowledge Seed": "知識種子",
+    "Last Teatime": "最後的下午茶",
+    "Manica of Control": "控制的護臂",
+    "Manifu": "瑪尼普",
+    "Merciless Glutton": "無情的大食客",
+    "Ms. Confille": "Miss Confile",
```

Comment:
Couple missing artifacts

Reply:
some artifact are not translated in the game like junkyard dog. 
![Screenshot_2021-03-07-23-48-02-018_com.stove.epic7.google.jpg](https://user-images.githubusercontent.com/734749/110257775-9a49fd80-7f9f-11eb-9f14-a5b84eb5f3c4.jpg)